views/email_preview_template.ejs

```diff
@@ -0,0 +1,225 @@
+<% include _header.ejs %>
+
+<div class="container">
+    <div class="col-lg-2"></div>
+
+    <div class="col-lg-8 email-preview-container">
+        
+        <!-- EMAIL PREVIEW TEMPLATE STARTS  -->
+
+        <div class="col-lg-12">
+            <h3 class="email-preview-title">EMail Preview Mode</h3>
+        </div>
+
+        <div class="col-lg-12 email-preview-fields-container">
+
+            <div class="form-group">
+                <label class="control-label col-lg-2" for="email-to">To</label>
+
+                <div class="col-lg-10 email-to">
+                  <label class="control-label field-input" for="email-to"> <%= allUserIds[participantCount].user.get("emails")[0].value %> </label>
+                </div>
+
+            </div>
+
+            <div class="form-group">
+                <label class="control-label col-lg-2" for="email-from">From</label>
+
+                <div class="col-lg-10 email-from">
+                   <label class="control-label field-input" for="email-from">srishti@media.mit.edu</label>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="control-label col-lg-2" for="email-sub">Subject</label>
+
+                <div class="col-lg-10 email-sub">
+                   <label class="control-label field-input" for="email-sub">Unhangout: Event Followup</label>
+                </div>
+            </div>
+
+            <table border="0" cellpadding="0" cellspacing="0">
+            <tr>
+
+            <td align="left" valign="top" style="border: 30px solid #f2f2f2" >
+
+            <div style="padding-top: 10px; padding-left: 10px; padding-right: 10px; padding-bottom: 10px; margin-left: 10px; margin-right: 10px; background-color: white"> 
+
+            <div class="email-header">
+
+                <h1 style="margin: 0px; font-size:24px;"><%= eventTitle %></h1>
+
+                <h2 style="font-size:18px">
+                Dear <%= allUserIds[participantCount].user.getShortDisplayName() %>,
+                <br />
+                </h2>
+
+                <p style="font-size:14px">You recently participated in the <b><%= eventTitle %></b>
+                Unhangout along with other people from all around the world! 
+
+                <% if(allUserIds[participantCount].users.length > 0) { %>  
+                    <br/><br/>
+                    Say hello to people you interacted with in the event on their favored method of contact. People who you <strong>chatted with</strong> in the <strong>breakout</strong>:<br />
+                <% } %>
+
+            </div>
+
+            <div class="breakout-info">
+
+                <table border="0" cellpadding="0" cellspacing="0" width="122">
```

Comment:
This should not be duplicated, at all.  It needs to be in a separate template and included.